src/components/20-molecules/datepicker/index.js

```diff
@@ -0,0 +1,369 @@
+      cells: { type: Array },
+      allowedYears: { type: Array },
+      labelButtonCancel: { type: String },
+      labelButtonOk: { type: String },
+    };
+  }
+
+  // TODO: set the selectd item of the year and month array from startdate
+  constructor() {
+    super();
+    this.locale = 'de-ch'; // TODO get from attributes
+    this.labelButtonCancel = 'Cancel'; // TODO get from attributes
+    this.labelButtonOk = 'OK'; // TODO get from attributes
+    this.startDate = new Date(); // TODO get from attributes
+    this.date = this.startDate;
+    this.outputDate = '';
+    this.year = this.date.getFullYear();
+    this.month = this.date.getMonth();
+    this.allowedYears = [2019, 2020]; // TODO get from attributes
+    this.open = false;
+
+    this.monthItems = getAllLocaleMonthsArray(this.locale).map((item, index) => ({
+      isSelected: index === this.month,
+      name: item.toString(),
+      value: index.toString(),
+    }));
+
+    this.yearItems = this.allowedYears.map(item => ({
+      isSelected: item === this.year,
+      name: item.toString(),
+      value: item.toString(),
+    }));
+
+    this.store = new Store(this.locale, this.startDate, this.allowedYears);
+    this.cells = this.store.getCells();
+    this.weekdays = getWeekdays(this.startDate, this.locale);
+  }
+
+  debounce(func, wait, immediate) {
+    let timeout;
+    return (...args) => {
+      const context = this;
+      const later = () => {
+        timeout = null;
+        if (!immediate) func.apply(context, args);
+      };
+      const callNow = immediate && !timeout;
+      clearTimeout(timeout);
+      timeout = setTimeout(later, wait);
+      if (callNow) func.apply(context, args);
+    };
+  }
+
+  firstUpdated() {
+    this.dropdownMonth = this.shadowRoot.querySelector('.js-datepicker__dropdown-month');
+    this.dropdownYear = this.shadowRoot.querySelector('.js-datepicker__dropdown-year');
+    this.inputField = this.shadowRoot.querySelector('.js-datepicker__input');
+    this.dropdownMonth.addEventListener('AXA_CHANGE', e => this.handleChangeDropdownMonth(e));
+    this.dropdownYear.addEventListener('AXA_CHANGE', e => this.handleChangeDropdownYear(e));
+    window.addEventListener('keydown', e => this.handleWindowKeyDown(e));
+    window.addEventListener('click', e => this.handleBodyClick(e));
+    window.setTimeout(() => {
+      window.addEventListener('resize', this.debounce(() => this.handleViewportCheck(this.inputField), 250));
+      window.addEventListener('scroll', this.debounce(() => this.handleViewportCheck(this.inputField), 250));
+      this.handleViewportCheck(this.inputField);
+    }, 100);
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+    window.axaComponents = window.axaComponents || {};
+    this.addEventListener('click', e => this.handleDatepickerClick(e));
+  }
+
+  disconnectedCallback() {
+    this.dropdownMonth.removeEventListener('AXA_CHANGE', e => this.handleChangeDropdownMonth(e));
```

Comment:
dtto

Reply:
did u run `npm run test` or just the linter? pretty sure that `e => ...` is not allowed. Instead i think `(e) => ..` is required from the linter. not sure thought

Reply:
prettier auto formats it to this. we have to change this anyways. This should be done in a separate PR.

Reply:
but if current ESLint requires it, u have to be eslint compliant in this PR

Reply:
Fixed all lint errors of datepicker and/or sub elements (like dropdown). 
(There are 3 lints errors open but not related to this PR)